templates/tfengine/components/cicd/auto/main.tf

```diff
@@ -0,0 +1,43 @@
+# Copyright 2020 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This folder contains additional Terraform resources in the devops project to configure CI/CD which
+# are also managed by CI/CD itself.
+#
+# Note that, the resources to put here should be carefully reviewed as the devops project is
+# sensitive and most changes should done by a human operator to avoid CI/CD misconfigures itself.
```

Comment:
This comment seems to contradict the notion of having an 'auto' deployment

Reply:
On second read this is fine.

Out of curiosity, can we add this auto deployment inside org?

Reply:
Added more comments. Let me know if that's more clear.

Reply:
This is actually under org/ in the final generated TF configs.